misc/python/materialize/cli/ci_logged_errors_detect.py

```diff
@@ -222,6 +222,16 @@ def get_known_issues_from_github() -> list[KnownIssue]:
 
     issues_json = response.json()
     assert issues_json["incomplete_results"] == False
+    return issues_json
+
+
+def get_known_issues_from_github() -> list[KnownIssue]:
+    page = 1
+    issues_json = get_known_issues_from_github_page(page)
+    while issues_json["total_count"] > len(issues_json["items"]):
+        page += 1
+        next_page_json = get_known_issues_from_github_page(page)
```

Comment:
Just to be safe, please put an early termination if `next_page_json` did not yield any new `"items"`.